templates/terraform/examples/cloud_run_domain_mapping_basic.tf.erb

```diff
@@ -0,0 +1,13 @@
+resource "google_cloudrun_domain_mapping" "<%= ctx[:primary_resource_id] %>" {
+  location = "us-central1"
+  provider = "google-beta"
+  name = "<%= ctx[:vars]['cloudrun_domain_name'] %>"
```

Comment:
Looks like this generated an empty name: https://github.com/terraform-providers/terraform-provider-google-beta/pull/757/files#diff-2b750590a17b245bd0d6188d7ed172b3